packages/@aws-cdk/aws-codepipeline/lib/pipeline.ts

```diff
@@ -255,7 +285,45 @@ export class Pipeline extends cdk.Construct implements cpapi.IPipeline {
 
   // ignore unused private method (it's actually used in Stage)
   // @ts-ignore
-  private _generateOutputArtifactName(stage: cpapi.IStage, action: cpapi.Action): string {
+  private _attachAction(stage: Stage, action: actions.Action): void {
```

Comment:
Since this is all about cross region support, I would name this something like `_attachActionToRegion`

Reply:
Changed.